varinfo/cf_config.py

```diff
@@ -125,8 +125,8 @@ class object.
         mission_matches = re.match(mission, self.mission) is not None
 
         short_name_matches = (
-            short_name is None or
-            re.match(short_name, self.short_name) is not None
+            short_name is None
+            or re.match(short_name, self.short_name) is not None
```

Comment:
The two changes in this file are styling things relating to the binary operator breaking stuff (W503 vs W504).

Reply:
Oh nice! So to test if our code adheres to style requirements we would do something `pycodestyle cmr_search.py` or can we just run `pycodestyle` in the root directory and it will check everything?

Reply:
The style check is now folded into the main test suite, so if you run `make test` it will run `pycodestyle` over the whole `varinfo` directory as part of that command. That's how I found these stragglers. Having that test in place now means that we can't merge the PR until the code is all compliant according to `pycodestyle` because that will be a test failure.

If you wanted to run it separately, though, you could use `pycodestyle` in the command line, too. `pycodestyle varinfo/*py` or `pycodestyle varinfo/cmr_search.py` as you say.